sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -494,7 +501,16 @@ private synchronized void setInstanceLogUrl(final String logUrl) {
 
                 @Override
                 public void run() {
-                    mChannel.setLogUrl(logUrl);
+                    if (mChannel != null) {
```

Comment:
mChannel is not null, please remove if, 

Some changes need to be done in AppCenter to delay setLogUrl in the channel to when we start. Keep logurl in appcenter class if called before start and apply it at start.

Reply:
Resolved in new commit.